NEMO/templates/consumables/consumables.html

```diff
@@ -123,6 +137,7 @@ <h1>Withdraw consumables</h1>
 		}
         window.addEventListener("load", function ()
         {
+			$('[data-toggle="notes-tooltip"]').tooltip();
```

Comment:
<br> are showing in tooltip
```suggestion
			$('[data-toggle="notes-tooltip"]').tooltip({"html": true});
```